contrib/objective-C/KalmanFilter.m

```diff
@@ -0,0 +1,77 @@
+/**
+ * Simple implementation of the Kalman Filter for 1D data.
+ * Originally written in JavaScript by Wouter Bulten
+ *
+ * Now rewritten into Objective-C
+ * 2018
+ *
+ * @license MIT License
+ *
+ * @author Sifan Ye
+ * @author Andreas Eppler
+ * @author Biraj Dhakal
```

Comment:
Only your name is enough here. All versions are derived from the base JS version.